### PR TITLE
ElytraFly improvements

### PR DESCRIPTION
**Pull request by cattyngmd**

### CODE REVIEW

**Comment by seasnale** [comment]
cattyn pr 🙀

**Comment by cattyngmd** [comment]
🙀

File: src/main/java/meteordevelopment/meteorclient/systems/modules/movement/elytrafly/ElytraFly.java [hunk 1]
```diff
@@ -354,6 +378,10 @@ private void onPlayerMove(PlayerMoveEvent event) {
         }
     }
 
+    public boolean canPacketEfly() {
+        return isActive() && flightMode.get() == ElytraFlightModes.Packet && mc.player.getEquippedStack(EquipmentSlot.CHEST).getItem() instanceof ElytraItem && !mc.player.isOnGround();
+    }
```
Comment (ThebestkillerTBK) [inline src/main/java/meteordevelopment/meteorclient/systems/modules/movement/elytrafly/ElytraFly.java:383]: glitchy 💀

Comment (cattyngmd) [reply src/main/java/meteordevelopment/meteorclient/systems/modules/movement/elytrafly/ElytraFly.java:383]: 😃 😃 😃

Comment (MineGame159) [reply src/main/java/meteordevelopment/meteorclient/systems/modules/movement/elytrafly/ElytraFly.java:383]: Why only packet mode?

**Pull request merged**